src/views/InProgress.vue

```diff
@@ -27,52 +27,52 @@
           </ion-label>
           <ion-icon :icon="printOutline" />
         </ion-item>
-      </div> 
-
+      </div>
       <ion-button expand="block" class="desktop-only" fill="outline" @click="packOrdersAlert">{{ $t("Pack orders") }}</ion-button>
-
-      <ion-card>
+      <ion-card v-for="(orders, index) in inProgressOrders.list.inProgress" :key="index">
         <div class="card-header">
           <div class="order-primary-info">
             <ion-label>
-              Darooty Magwood
-              <p>{{ $t("Ordered") }} 27th January 2020 9:24 PM EST</p>
+              {{ orders.doclist.docs[0].customerName }}
+              <p>{{ $t("Ordered") }} {{ $filters.formatUtcDate(orders.doclist.docs[0].orderDate, 'YYYY-MM-DDTHH:mm:ssZ', 'Do MMMM YYYY LT z') }}</p>
             </ion-label>
           </div>
 
           <div class="order-tags">
             <ion-chip outline>
               <ion-icon :icon="pricetagOutline" />
-              <ion-label>NN10584</ion-label>
+              <ion-label>{{ orders.doclist.docs[0].orderId }}</ion-label>
             </ion-chip>
           </div>
 
-          <div class="order-metadata">
+          <!-- <div class="order-metadata">
```

Comment:
Uncomment this div and instead comment the `ion-label`, as commenting this div will distort the UI.

Reply:
Done Sir